.flake8

```diff
@@ -1,2 +1,2 @@
 [flake8]
-ignore = E501, W503, W504
+ignore = E501, W503, W504, W605
```

Comment:
I think I'd prefer to avoid to ignore W605 here.
Putting a 'r' before the string in regression.py fixes the issue:
r'[\[]?Feature/Bug causing the regression[....